### PR TITLE
Add new package: ima-evm-utils

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/ima-evm-utils/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from spack import *
+
+
+class ImaEvmUtils(AutotoolsPackage):
+    """IMA/EVM control utilities."""
+
+    homepage = "http://linux-ima.sourceforge.net"
+    url      = "http://sourceforge.net/projects/linux-ima/files/ima-evm-utils/ima-evm-utils-1.2.1.tar.gz"
+
+    version('1.3.1', sha256='5304271f31a3601a2af5984942d9bd6c7532597c5a97250c9a4524074fc39925')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ima-evm-utils/package.py:16]: Curious: Why pick the least downloaded versions of the package?  Why not include the latest?

File: var/spack/repos/builtin/packages/ima-evm-utils/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class ImaEvmUtils(AutotoolsPackage):
+    """IMA/EVM control utilities."""
+
+    homepage = "http://linux-ima.sourceforge.net"
+    url      = "http://sourceforge.net/projects/linux-ima/files/ima-evm-utils/ima-evm-utils-1.2.1.tar.gz"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ima-evm-utils/package.py]: Please use `https` for the url and, ideally, for the homepage when possible.

```suggestion
    url      = "https://sourceforge.net/projects/linux-ima/files/ima-evm-utils/ima-evm-utils-1.2.1.tar.gz"
```

Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ima-evm-utils/package.py]: It looks like there's access to more versions, etc if you use the repository at https://sourceforge.net/p/linux-ima/ima-evm-utils/ci/master/tree/.

**Comment by darmac** [comment]
OK, I have a lot of packages which out of date for now(prepared to make PR in last year but not).

**Comment by darmac** [comment]
Is that patch OK for merging? @tldahlgren

**Review by tldahlgren — APPROVED** [review]

**Comment by tldahlgren** [comment]
> Is that patch OK for merging? @tldahlgren

Thank you for making the requested changes.  I'm still curious about versions but have gone ahead and approved your PR.

**Pull request merged**